File: Civitas - Java/src/GUI/JuegoTexto.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package GUI;
import civitas.CivitasJuego;
import java.util.ArrayList;
import java.util.Scanner;
import java.util.Arrays;

/**
 *
 * @author juanc
 */
public class JuegoTexto {
    
    public static void main(String[] args) {
        VistaTextual vista = new VistaTextual();
        CivitasJuego civitas;
        Controlador controlador;
        Scanner in = new Scanner(System.in);
        int numJugadores;
        
        ArrayList<String> nombres = new ArrayList<>();
        
        System.out.println("¿Cuantos jugadores quieres?");
        numJugadores = in.nextInt();
        in.nextLine();
        
        if( numJugadores >= 2 && numJugadores <=4 ){
            for(int i = 0; i < numJugadores; i++){
                System.out.println("Nombre del juagdor " + i + ": ");
                nombres.add( in.nextLine() );
            }
            civitas = new CivitasJuego(nombres);
            Dado.getInstance().setDebug(false);
//            controlador = new Controlador(civitas, vista);  
//            controlador.juega();
        }
        else System.err.println("Numero de jugadores erroneo");
    }
}

File: Civitas - Ruby/lib/juego_texto.rb
# To change this license header, choose License Headers in Project Properties.
# To change this template file, choose Tools | Templates
# and open the template in the editor.

require './vista_textual'
require './Controlador'
require './jugador_especulador'

module Civitas

    puts "Comienzo de programa............."
    vista = Vista_textual.new()
    nombres = []
    
    puts "Cuantos jugadores quieres?"
    num_jugadores = gets.to_i 
  
    if( num_jugadores >= 2 && num_jugadores <=4 )
       for i in 0..(num_jugadores-1)
           puts("Nombre del juagdor #{i} : ")
           nombres.push( gets.chomp )
       end
        civitas = Civitas_juego.new(nombres)
        Dado.instance.set_debug(true)
        controlador = Controlador.new(civitas, vista); 
        controlador.juega
  
    else puts("Numero de jugadores erroneo. FIN DEL PROGRAMA")
    end

end
